apollo-client/src/main/java/com/ctrip/framework/apollo/spring/property/AutoUpdateConfigChangeListener.java

```diff
@@ -39,18 +39,16 @@
   private static final Logger logger = LoggerFactory.getLogger(AutoUpdateConfigChangeListener.class);
 
   private final boolean typeConverterHasConvertIfNecessaryWithFieldParameter;
-  private final Environment environment;
   private final ConfigurableBeanFactory beanFactory;
   private final TypeConverter typeConverter;
   private final PlaceholderHelper placeholderHelper;
   private final SpringValueRegistry springValueRegistry;
   private final Gson gson;
 
-  public AutoUpdateConfigChangeListener(Environment environment, ConfigurableListableBeanFactory beanFactory){
```

Comment:
This is a public API and it resides in client module, so this is potentially a breaking change, though the users are rarely possible to use it, what do other reviewers' opinion ?

Reply:
I also agree we should be very very careful when introducing potentially breaking changes.
I checked the commit history and the environment is not used since #2173. But since this is a public class, I think it's better not to change the constructor signature.